docs/reference/argocd.md

```diff
@@ -847,6 +848,12 @@ LogFormat | text | The log format to be used by the ArgoCD Repo Server. Valid op
 ExecTimeout | 180 | Execution timeout in seconds for rendering tools (e.g. Helm, Kustomize)
 Env | [Empty] | Environment to set for the repository server workloads
 Replicas | [Empty] | The number of replicas for the ArgoCD Repo Server. Must be greater than or equal to 0.
+Volumes | [Empty] | Add addition volumes to the repo server deployment.
```

Comment:
Could be reworded as `Provision additional volumes`  or `Configure additional volumes`

Reply:
reworded to `Configure additional...`